src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -146,6 +178,32 @@ const ExerciseFinishedScreen = ({ navigation, route }: ExerciseFinishedScreenPro
       backgroundColor={unlockedNextExercise ? theme.colors.correct : theme.colors.primary}
       lightStatusBarContent={!unlockedNextExercise}
       bottomBackgroundColor={theme.colors.background}>
+      {exercise === EXERCISES.length - 1 ? (
+        <ModalSkeleton visible={isModalVisible} onClose={() => setIsModalVisible(false)}>
```

Comment:
You do not need to write you own modal, you can use our "Modal.tsx" component, where you only need to pass the correct props. You need to adjust it a bit to get the icon above the heading text, but this can easily be done by adding an optional icon prop to to pass the icon to the Modal component.

Reply:
@SteffiStuffel is it possible to insert two icons and another button `später`? I think I need to add these options to "Modal.tsx" or ?

Reply:
You can adjust the modal, so you can pass two icons and also the text for the cancelButton (as it is currently only done for the confirmation button)